x-pack/docs/en/security/authorization/document-level-security.asciidoc

```diff
@@ -3,19 +3,30 @@
 === Document level security
 
 Document level security restricts the documents that users have read access to.
-In particular, it restricts which documents can be accessed from document-based 
-read APIs. 
+In particular, it restricts which documents can be accessed from document-based
+read APIs.
 
 To enable document level security, you use a query to specify the documents that
-each role can access. The document query is associated with a particular data
+each role can access. The document `query` is associated with a particular data
 stream, index, or wildcard (`*`) pattern and operates in conjunction with the
 privileges specified for the data streams and indices.
 
+The specified document `query`:
+
+* Expects the same format as if it was defined in the search request
+* Supports <<templating-role-query,templating>> for `_user` variables
```

Comment:
We could talk about the actual info that can be accessed instead of just giving out a variable name, something like:
```
* Supports <<templating-role-query,templating>> that can access the details of the current authenticated user
```